### PR TITLE
L2ListenTcpdump windows

### PR DESCRIPTION
**Pull request by gpotter2**
fix https://github.com/secdev/scapy/issues/2496

### CODE REVIEW

File: scapy/supersocket.py [hunk 1]
```diff
@@ -389,6 +390,10 @@ def close(self):
         SuperSocket.close(self)
         self.tcpdump_proc.kill()
 
+    @staticmethod
```
Comment (p-l-) [inline scapy/supersocket.py]: Why that? This is going to have massive performance impacts. Or am I missing something?

Comment (gpotter2) [reply scapy/supersocket.py]: This should now only impact windows and darwin, on which the select call wouldn't work anyways

**Comment by gpotter2** [comment]
There is a commented unit test in the code, but it was too unstable so we disabled it.
https://github.com/secdev/scapy/blob/b21e2c5fe5464fe7102f2c427d9a13d8b49f3e10/test/regression.uts#L1758
If i recall it's pretty glitchy.. especially on pypy

**Comment by gpotter2** [comment]
There was a unit test but we disabled it because it was too unstable. If i recall it was especially glitchy on PyPy...

https://github.com/secdev/scapy/blob/b21e2c5fe5464fe7102f2c427d9a13d8b49f3e10/test/regression.uts#L1755-L1775

**Comment by p-l-** [comment]
@guedou LGTM, your call.

**Pull request merged**